pkg/testing/runner/debian.go

```diff
@@ -147,7 +147,11 @@ func (DebianRunner) Run(ctx context.Context, c *ssh.Client, logger Logger, agent
 		vars := fmt.Sprintf(`GOPATH="$HOME/go" PATH="$HOME/go/bin:$PATH" AGENT_VERSION="%s" TEST_DEFINE_PREFIX="%s" TEST_DEFINE_TESTS="%s"`, agentVersion, prefix, strings.Join(tests, ","))
 		vars = extendVars(vars, env)
 		logger.Logf("Starting tests")
-		script := fmt.Sprintf(`cd agent && %s ~/go/bin/mage integration:testOnRemote`, vars)
+		logArg := ""
+		if mg.Verbose() {
+			logArg = "-v"
+		}
```

Comment:
I don't know when `mg` was imported or added, maybe I missed it in a previous PR. But that should not be imported into the `pkg/testing`. It was designed for other projects to use and has no requirement on the usage of `mage`.

Please look at passing in verbose flag to the `pkg/testing/runner` configuration. You can set that configuration flag in our `magefile.go`.

Reply:
Ah, you're right! Didn't notice we were kinda outside the scope of the mage packages.